src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -928,9 +974,16 @@ bool clang_cpp_convertert::get_decl_ref(
 
   default:
   {
-    msg.error(fmt::format(
-      "Conversion of unsupported clang decl ref: {}", decl.getDeclKindName()));
-    abort();
+    // If there is an unsupported clang decl, first try to forward it to clang_c_convertert::get_decl_ref.
+    // It might be already handled there.
```

Comment:
> The code below prints an error and does not forward to clang_c_convertert::get_decl_ref() 

The forwarding happens in line 954 - 957. The comment here hints that when we see this error message extending the cases in line 954 might resolve the issue. 

Here I'm coding in a defensive way. I don't want to blindly put the forwarding in `default` because I would like to explicitly distinguish `clang::decl::Foo` that can be handled by forwarding from `clang::decl::CXXFoo` that requires additional parsing.

Reply:
I like this comment. Done. Added it in the code.